b2sdk/utils/thread_pool.py

```diff
@@ -8,38 +8,17 @@
 #
 ######################################################################
 
-from concurrent.futures import ThreadPoolExecutor, Future
-from threading import Lock
-from typing import Optional, Callable
+from concurrent.futures import ThreadPoolExecutor
+from typing import Optional
 
 from b2sdk.utils import B2TraceMetaAbstract
 
 
-class LazyThreadPool(metaclass=B2TraceMetaAbstract):
+class ThreadPoolMixin(metaclass=B2TraceMetaAbstract):
     """
-    Lazily initialized thread pool.
-    Can be safely used between threads.
+    Mixin class with lazily initialized ThreadPoolExecutor. Threadsafe.
```

Comment:
is this one lazy though?

Reply:
Nope, but it was before the last refactor.

Reply:
Now it's just a mixin with a thread pool.

Reply:
I think I deleted too much, it's no longer threadsafe as well. :( Will fix.

Reply:
Fixed in [fc9de68](https://github.com/Backblaze/b2-sdk-python/pull/297/commits/fc9de68873550ac431cee0d257bb714d1ebee152)